wiki/Article_styling_criteria/Formatting/th.md

```diff
@@ -0,0 +1,788 @@
+
+```markdown
+*หน้าหลัก: {บทความ}*
+
+*หน้าหลัก: {บทความ} และ {บทความ}*
+```
+
+### ดูสิ่งนี้ด้วย
+
+บันทึกย่อ *ดูสิ่งนี้ด้วย* แนะนำให้ผู้อ่านจุดที่น่าสนใจอื่น ๆ จากบทความหรือส่วนที่กำหนด ต้องจัดรูปแบบดังนี้:
+
+```markdown
+*ดูเพิ่มเติมที่: {บทความ}*
+
+*ดูเพิ่มเติมที่: {บทความ} และ {บทความ}*
+```
+
+### สำหรับดู
+
+บันทึกย่อ *สำหรับดู* คล้ายกับหมายเหตุหมวก *ดูสิ่งนี้ด้วย* แต่โดยทั่วไปมีคำอธิบายและตรงไปตรงมามากกว่า บันทึกย่อ นี้อาจใช้มากกว่าหนึ่งลิงก์หากจำเป็น ต้องจัดรูปแบบดังนี้:
+
+```markdown
+*สำหรับ {คำอธิบาย} โปรดดูที่: {บทความ}*
+
+*สำหรับ {description} ดู: {article} และ {article}*
+```
+### เพื่อไม่ให้สับสนกับ
+
+บันทึกย่อ *เพื่อไม่ให้สับสนกับ* ช่วยแยกแยะชื่อหรือหัวข้อบทความที่คลุมเครือหรือเข้าใจผิด บันทึกย่อ นี้อาจใช้มากกว่าหนึ่งลิงก์หากจำเป็น ต้องจัดรูปแบบดังนี้:
+
+```markdown
+*อย่าสับสนกับ {บทความ}.*
+
+*อย่าสับสนกับ {บทความ} หรือ {บทความ}.*
+```
+
+### สำหรับการใช้งานอื่นๆ
+
+บันทึกย่อ *สำหรับการใช้งานอื่นๆ* จะคล้ายกับ *เพื่อไม่ให้สับสนกับ* บันทึกย่อ แต่มีการเชื่อมโยงโดยตรงไปยัง[บทความแก้ความกำกวม](#disambiguation-articles) หมวกใบนี้ต้องลิงก์ไปยังบทความแก้ความกำกวมเท่านั้น ต้องจัดรูปแบบดังนี้:
+
+```markdown
+*สำหรับความหมายอื่น ดูที่ {บทความแก้ความกำกวม}*
+```
+
+## หมายเหตุ
+
+*เพื่อไม่ให้สับสนกับ [บันทึกย่อ](#บันทึกย่อ)*
+
+ควรมีการแจ้งเตือนในส่วนที่เหมาะสม แต่ต้องเริ่มจากย่อหน้าและใช้ตัวเอียง ประกาศอาจมีตัวหนาตามความเหมาะสม แต่ควรเก็บไว้ให้น้อยที่สุด คำบอกกล่าวจะต้องเขียนเป็นประโยคที่สมบูรณ์ ดังนั้น จึงไม่เหมือนกับ [บันทึกย่อย่อ](#บันทึกย่อ) ส่วนใหญ่ ต้องใช้จุดเต็ม (`.`) หรือเครื่องหมายอัศเจรีย์ (`!`) หากเหมาะสม สิ่งใดก็ตามที่อยู่ในวรรคเดียวกันของประกาศจะต้องเป็นตัวเอียงด้วย ต้องจัดรูปแบบดังนี้:
+
+```markdown
+*หมายเหตุ: {หมายเหตุ}.*
+
+*ประกาศ: {ประกาศ}.*
+
+*ข้อควรระวัง: {ข้อควรระวัง}.*
+
+*คำเตือน: {คำเตือน}.*
+```
+
+- ควรใช้ `หมายเหตุ' สำหรับรายละเอียดที่เป็นข้อเท็จจริงหรือเล็กน้อย
+- ควรใช้ "ประกาศ" เพื่อเตือนความจำหรือดึงความสนใจไปยังสิ่งที่ผู้อ่านควรทราบ
+- ควรใช้ "ข้อควรระวัง" เพื่อเตือนผู้อ่านเพื่อหลีกเลี่ยงผลกระทบที่ไม่ได้ตั้งใจ
+- ควรใช้ "คำเตือน" เพื่อเตือนผู้อ่านว่าอาจมีการดำเนินการกับพวกเขา
+
+## เน้น
+
+### ตัวหนา
+
+ตัวหนาต้องใช้เครื่องหมายดอกจันคู่ (`**`)
+
+ย่อหน้านำอาจทำให้ชื่อบทความปรากฏขึ้นครั้งแรกเป็นตัวหนา
+
+### ตัวเอียง
+
+ตัวเอียงต้องใช้ดอกจันเดี่ยว (`*`)
+
+ชื่องานหรือวิดีโอเกมควรเป็นตัวเอียง osu!—เกม—ได้รับการยกเว้นจากสิ่งนี้
+
+การเกิดขึ้นครั้งแรกของตัวย่อ ตัวย่อ หรืออักษรย่ออาจเป็นตัวเอียง
+
+อาจใช้ตัวเอียงเพื่อเน้นหรือช่วยในการอ่านได้
+
+## หัวเรื่อง
+
+หัวเรื่องทั้งหมดต้องใช้ตัวพิมพ์ประโยค
+
+ส่วนหัวต้องใช้รูปแบบ [ATX (hash)](https://github.github.com/gfm/#atx-headings "GitHub") และต้องมีบรรทัดว่างก่อนและหลังส่วนหัว หัวเรื่องเป็นข้อยกเว้นเมื่ออยู่ในบรรทัดแรก หากเป็นกรณีนี้ จะต้องเว้นบรรทัดว่างไว้หลังหัวเรื่องเท่านั้น
+
+หัวเรื่องต้องไม่เกินระดับหัวเรื่อง 5 และห้ามใช้เพื่อจัดรูปแบบหรือจัดรูปแบบข้อความ
+
+### ชื่อเรื่อง
+
+*See also: [การตั้งชื่อบทความ](#การตั้งชื่อบทความ)*
+
+*ข้อควรระวัง: ชื่อจะถูกแยกวิเคราะห์เป็นข้อความธรรมดา พวกเขาจะต้องไม่หลบหนี*
```

Comment:
```suggestion
*ข้อควรระวัง: ชื่อจะถูกแยกวิเคราะห์เป็นข้อความธรรมดา พวกเขาจะต้องไม่ถูก escape*
```

Reply:
I am too lazy that i translate some lines with google